react/components/OrganizationsAutocomplete.tsx

```diff
@@ -11,7 +11,7 @@ const initialState = {
   status: ['active', 'on-hold', 'inactive'],
   search: '',
   page: 1,
-  pageSize: 25,
+  pageSize: 999,
```

Comment:
Instead of setting the page size to 999, can't we handle the pagination? Also, have we tested this with 999 cost centers to check the behaviour?

Reply:
decrease the page size to 500